datasets/snow-projections-diff.data.mdx

```diff
@@ -11,7 +11,14 @@ media:
 taxonomy:
   - name: Topics
     values:
```

Comment:
where did topic values here come from. Much more specific that others which leads to inaccurate/confusing faceted search results.